Source/JavaScriptCore/wasm/WasmOperationsInlines.h

```diff
@@ -131,7 +135,9 @@ JSWebAssemblyArray* copyElementsInReverse(JSWebAssemblyInstance* instance, Wasm:
     JSGlobalObject* globalObject = instance->globalObject();
     VM& vm = instance->vm();
 
-    auto* array = JSWebAssemblyArray::create(vm, globalObject->webAssemblyArrayStructure(), fieldType, size, WTFMove(rtt));
+    auto* array = JSWebAssemblyArray::tryCreate(vm, globalObject->webAssemblyArrayStructure(), fieldType, size, WTFMove(rtt));
```

Comment:
Should we rename `copyElementsInReverse` to `tryCopyElementsInReverse` ?

Reply:
Ditto.